sound/soc/sof/ops.h

```diff
@@ -201,8 +201,8 @@ snd_sof_dsp_set_power_state(struct snd_sof_dev *sdev,
 	if (sof_ops(sdev)->set_power_state)
 		return sof_ops(sdev)->set_power_state(sdev, target_state);
 
-	/* D0 substate is not supported */
-	return -ENOTSUPP;
+	/* D0 substate is not supported, so ignore it. */
+	return 0;
```

Comment:
@xiulipan can fix the comment to say that nothing to do if set power state op is not set? Also typo in the commit message "sate"